warehouse/query-core/src/main/java/datawave/query/tables/edge/EdgeQueryLogic.java

```diff
@@ -699,14 +699,12 @@ public void setQueryThreads(int queryThreads) {
     
     @Override
     public Set<String> getOptionalQueryParameters() {
-        Set<String> params = new TreeSet<>();
-        params.add(datawave.webservice.query.QueryParameters.QUERY_BEGIN);
```

Comment:
It looks like begin and end date are optional for EdgeQuery (I'm looking at EdgeQuery.getDateFilter).

Reply:
Good catch